highfidelity/hifi
# Mouse cursor should not visible in HMD Screen in HMD Mode

Fix for 
1) case 3844 : Don-t-use-computer-cursor-in-VR-mode : https://highfidelity.fogbugz.com/f/cases/3844/Don-t-use-computer-cursor-in-VR-mode
2) case 4389 : Interface-takes-over-mouse-control : https://highfidelity.fogbugz.com/f/cases/4389/Interface-takes-over-mouse-control
3) case 3009 : Cursor-sometimes-follows-HMD-gaze-position-sometimes-doesn't : https://highfidelity.fogbugz.com/f/cases/3009/Cursor-sometimes-follows-HMD-gaze-position-sometimes-doesn-t


**HMD required**

# How To Test

1. Open Interface window , switch to HMD mode.
2. Observe that, you will not see mouse in HMD, but mouse will visible in Desktop screen and you will not able to click anywhere except top menu bar of interface window.
3. switch to Desktop mode, desktop mode will as work as it is.

**Expected Result :** 

1. In HMD mode, mouse should be visible to desktop users/spectators, but not be able to click anywhere except the top menu.
2. Mouse cursor should not be visible on HMD screen.
3. Smooth transition of mouse when switching between Desktop and HMD modes.


**Note:** Dev tested on Windows 10 + Oculus Rift
